webapp/channels/src/packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -269,6 +269,59 @@ function formatPostInChannel(post: Post, previousPost: Post | undefined | null,
     };
 }
 
+export function getLatestPostIdToReact(state: GlobalState, channelId: string, rootId = '') {
```

Comment:
Does this replace `getLatestReplyablePostId` at all? If it does, perhaps we could remove that one

Reply:
Actually, I took another look and the code you have here was right because they're two different posts. In the old code for `AdvancedCreatePost`, we have two props: `latestReplyablePostId` which is used for replying/reacting and `currentUsersLatestPost` which is used for editing. Thinking about it more, that makes sense since you can reply/react to anyone's post, but you can only edit your own posts (unless you're an admin, but we don't want to pressing up to reply to those).

Given that, we do need both selectors, but this might still mean that we can get rid of the old selectors like `getLatestReplyablePostId` if they're not used any more

Reply:
`getLatestReplyablePostId` is still used (to handle hitting the up key). The one that doesn't seem to be used anymore is `getCurrentUsersLatestPost`, so I removed it.